packages/aws-cdk-lib/package.json

```diff
@@ -61,12 +59,11 @@
         "maven": {
           "groupId": "software.amazon.awscdk",
           "artifactId": "lib",
```

Comment:
```suggestion
          "artifactId": "lib"
```

Think this is what's breaking the build.

Reply:
Gah!!